### PR TITLE
lbAlert changes

### PR DESCRIPTION
**Pull request by Makdaam**
changed sev4->sev3 on availability alert
added degraded alert with <67 threshold over 1h
decreased threshold on availability alert to make it less sensitive than degraded alert

### CODE REVIEW

File: pkg/deploy/generator/templates.go [hunk 1]
```diff
@@ -63,7 +63,9 @@ func (g *generator) rpTemplate() *arm.Template {
 	}
 
 	if g.production {
-		t.Resources = append(t.Resources, g.pip(), g.lb(), g.vmss(), g.lbAlert())
+		t.Resources = append(t.Resources, g.pip(), g.lb(), g.vmss(),
+			g.lbAlert(50.0, 3, "rp-availability-alert", "PT5M", "PT1H"), // 3/3 backends down for 30min or 2/3 backends down for 45min in the last 1h
```
Comment (mjudeikis) [inline pkg/deploy/generator/templates.go:67]: I think if all 3 backends are dead for 30 min we might have a bigger problem and no need to wait for 30 min?

**Comment by mjudeikis** [comment]
lgtm.
rebase is needed

File: deploy/rp-production.json [hunk 2]
```diff
@@ -269,7 +311,7 @@
                     }
                 ]
             },
```
Comment (jim-minter) [inline deploy/rp-production.json:313]: please add the sdn check as well

File: deploy/rp-production.json [hunk 3]
```diff
@@ -249,9 +291,9 @@
                     "allOf": [
                         {
                             "operator": "LessThan",
```
[the remark this answers is not in the document]
Comment (Makdaam) [reply deploy/rp-production.json:293]: Turns out the allowed values are PT1H and then PT6H, increased to 6H and the frequency to 15.

**Comment by jim-minter** [comment]
please rebase, then will merge

**Pull request merged**